app/assets/javascripts/services/event_notifications_service.js

```diff
@@ -91,6 +91,11 @@ angular.module('ManageIQ').service('eventNotifications', ['$timeout', '$window',
           });
 
           updateUnreadCount(events);
+          var x = 0;
+          _.forEach(state.groups, function(group) {
```

Comment:
I think you don't need the `_` here, you [can](https://caniuse.com/#feat=mdn-javascript_builtins_array_foreach) directly call [`forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) on the array. Anyway, this code is a great example for using [`reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) on the array instead of having the helper variable.